### PR TITLE
Update EIP-7702: Warm Signers

### PR DESCRIPTION
**Pull request by wjmelements**
Reviewers @vbuterin @SamWilsn @adietrichs @lightclient
#### Changes
Start the signer accounts warm

### CODE REVIEW

**Review by SamWilsn — DISMISSED** [review]

File: EIPS/eip-7702.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ status: Draft
 type: Standards Track
 category: Core
 created: 2024-05-07
-requires: 2718, 2930
+requires: 2718, 2930, 2929
```
Comment (wjmelements) [inline EIPS/eip-7702.md:11]: @SamWilsn 2930 requires 2929 so this is redundant I think

Comment (wjmelements) [reply EIPS/eip-7702.md]: hmm 2930 also requires 2718

Comment (SamWilsn) [reply EIPS/eip-7702.md]: Yeah, that's fair. I mentioned it explicitly because I linked to it. There aren't any hard rules on transitive `requires` :rofl:

**Review by SamWilsn — APPROVED** [review]

**Comment by LwkkyLDdubz** [comment]
> ✅ All review

**Comment by LwkkyLDdubz** [comment]
All reviews read

**Pull request merged**